docs/source/library/Architecture.rst

```diff
@@ -0,0 +1,52 @@
+Architecture
+============
+
+The *architecture* describes the properties of the target device when mapping a quantum circuit.
+
+Constructing an Architecture
+############################
+
+An architecture can be constructed in various ways. The most convenient is to simply define the coupling map and pass it to the constructor of the Architecture class.
+
+    .. code-block:: python3
+
+        from mqt import qmap
+
+        coupling_map = set([(0, 1), (1, 2), (2, 3), (2, 4)])
```

Comment:
```suggestion
        coupling_map = {(0, 1), (1, 2), (2, 3), (2, 4)}
```
should work.

Reply:
why not use the same 4 qubit architecture here as in the other examples?

Reply:
No reason, I think. changed it to the linear architecture.